### PR TITLE
Integrate Dockerfile for easier builds

### PR DESCRIPTION
**Pull request by bobbysteel**
To enable easier Docker builds, integrate the Dockerfile and helper scripts.

The current linuxserver/medusa image is great, but builds aren't triggered as it only polls their linuxserver/docker-medusa repo which is rarely updated. This will enable auto rebuilding upon commits to master.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# install app
+COPY . /app/medusa/
+
+# copy local files
+COPY docker/root/ /
+
+# ports and volumes
+EXPOSE 8081
+VOLUME /config /downloads /tv
```
Comment (OmgImAlexis) [inline Dockerfile]: Would you mind adding a volume mount for anime since some people do like to keep them separate.

**Comment by OmgImAlexis** [comment]
We may want to move `docker` to `.docker` so we keep the root dir clean.

**Comment by bobbysteel** [comment]
@OmgImAlexis done!

**Comment by bobbysteel** [comment]
I'm open to that - anyone else have a view? I chose the docker/ folder
structure to mirror huginn which seems to use it successfully
https://github.com/huginn/huginn. But I'm not bothered either way really.

**Comment by bobbysteel** [comment]
Done now @OmgImAlexis

**Comment by medariox** [comment]
Can you rebase, please?

**Comment by labrys** [comment]
@bobbysteel can you change the `MAINTAINER` to a URL or email address?  Additionally, the `MAINTAINER` instruction is deprecated in favor of `LABEL maintainer`, see the [docs](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

**Comment by bobbysteel** [comment]
Sure - I can assign to myself or link a url to issues here on github - what
would you suggest?

**Pull request merged**